cmd/uhc/account/quota/cmd.go

```diff
@@ -120,20 +113,19 @@ func run(cmd *cobra.Command, argv []string) {
 		quotasListResponse, err := quotaClient.List().
 			Send()
 		if err != nil {
-			fmt.Fprintf(os.Stderr, "Failed to retrieve quota: %v\n", err)
-			os.Exit(1)
+			return fmt.Errorf("Failed to retrieve quota: %v", err)
 		}
 
 		// Display quota information:
-		fmt.Printf("Cluster quota for organization '%s' ID: '%s'\n",
+		fmt.Printf("Cluster quota for organization '%s' ID: '%s'",
```

Comment:
I think that here we need to preserve the `\n` because this is not returned as an error, just printed.

Reply:
Fixed!